Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
I don't think there's an advantage of `raise` over `odie` if the error output to users is the same. I don't think it's really necessary to be consistent but if we do I'd prefer we did on `odie` which is documented.